resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -777,35 +777,14 @@ function resize() {
             }
 
             if ($permissions >= Permissions::Developer) {
-                echo "<form class='mb-2' method='post' action='/request/game-relation/create.php'>";
-                echo csrf_field();
-                echo "<input type='hidden' name='game' value='$gameID'>";
-                echo "<div class='md:grid grid-cols-[180px_1fr_100px] gap-1 items-center mb-1'>";
-                echo "<label for='game_relation_add'>Add Related Games<br>(CSV of game IDs)</label>";
-                echo "<input type='text' name='relations' id='game_relation_add' class='w-full'>";
-                echo "<div class='text-right'><button class='btn'>Add</button></div>";
-                echo "</div>";
-                echo "</form>";
+                if ($userModel && $userModel->can('manage', [\App\Models\GameSet::class])) {
```

Comment:
Do you still need/want to check `Permissions::Developer` on the previous line?

Reply:
Nope - removed in latest.